pyproject.toml

```diff
@@ -83,6 +84,7 @@ sendgrid = []
 sendinblue = []
 sparkpost = []
 unisender-go = []
+scaleway = []
```

Comment:
Please keep these in alphabetical order. (I'm not sure why postal is at the end.)